src/main/java/net/snowflake/client/config/SFConnectionConfigParser.java

```diff
@@ -127,10 +124,23 @@ public static ConnectionParameters buildConnectionParameters() throws SnowflakeS
     }
   }
 
-  private static String createUrl(String account, Map<String, String> fileConnectionConfiguration) {
-    String host = String.format("%s.snowflakecomputing.com", account);
+  private static String createUrl(Map<String, String> fileConnectionConfiguration)
+      throws SnowflakeSQLException {
+    Optional<String> maybeAccount = Optional.ofNullable(fileConnectionConfiguration.get("account"));
+    Optional<String> maybeHost = Optional.ofNullable(fileConnectionConfiguration.get("host"));
```

Comment:
Should we verify if account matches to host? I'm not sure.

Reply:
Host should always override account parameter

Reply:
It's true. I confirmed that hostis preferable. I added only the warn with information about inconsistency.